tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/ConsoleMFLogger.cs

```diff
@@ -9,6 +9,10 @@ namespace MultiFunctionApplicationHelpers
     public static class ConsoleMFLogger
     {
 
+        private static string LogTs => DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss,fff");
+
+        private static int Tid = System.Threading.Thread.CurrentThread.ManagedThreadId;
```

Comment:
You probably want `=>` instead of `=` on this line.

Reply:
Good catch!  🤦